utils/node-loader/src/utils.rs

```diff
@@ -41,18 +53,33 @@ pub fn str_to_wsaddr(endpoint: String) -> WSAddress {
     WSAddress::new(domain, port)
 }
 
+pub fn iterator_with_args<T, F: FnMut() -> T>(
+    max_size: usize,
+    mut args: F,
+) -> impl Iterator<Item = T> {
+    let mut size = 0;
+    iter::from_fn(move || {
+        if size >= max_size {
+            return None;
+        }
+
+        size += 1;
+
+        Some(args())
+    })
+}
+
+pub fn convert_iter<V, T: Into<V>>(args: Vec<T>) -> impl IntoIterator<Item = V> {
+    args.into_iter().map(|t| t.into())
```

Comment:
A few more func-style 😄
```suggestion
    args.into_iter().map(Into::into)
```

Reply:
done in [8c58df2](https://github.com/gear-tech/gear/pull/1600/commits/8c58df269e07992ad196e1793a6fab68f63d0671)